this is removed in 1.0, but it's still needed by anything trying to find this
with autotools
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,10 @@
 option(GLM_ENABLE_CXX_17 "Enable C++ 17" OFF)
 option(GLM_ENABLE_CXX_20 "Enable C++ 20" OFF)
 
+set(GLM_PC ${CMAKE_CURRENT_BINARY_DIR}/glm.pc)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/glm.pc.in ${GLM_PC} @ONLY)
+install(FILES ${GLM_PC} DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
+
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if(GLM_ENABLE_CXX_20)
--- /dev/null
+++ b/glm.pc.in
@@ -0,0 +1,7 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/include
+
+Name: GLM
+Description: OpenGL Mathematics
+Version: @GLM_VERSION@
+Cflags: -I${includedir}
